conda_build_all/resolved_distribution.py

```diff
@@ -27,12 +29,18 @@ def setup_vn_mtx_case(case):
             conda_build.config.config.CONDA_PY = version
         elif pkg == 'numpy':
             conda_build.config.config.CONDA_NPY = version
+        elif pkg == 'perl':
```

Comment:
The whole function is untested directly, although it does get exercised by the integration tests.